README.md

```diff
@@ -7,7 +7,10 @@
 [![Project Status: Active - The project has reached a stable, usable state and is being actively developed.](http://www.repostatus.org/badges/latest/active.svg)](http://www.repostatus.org/#active)
 [![bitHound Code](https://www.bithound.io/github/tribou/jest-serializer-path/badges/code.svg)](https://www.bithound.io/github/tribou/jest-serializer-path)
 
-Remove absolute paths from your Jest snapshots.
+Remove absolute paths and normalize paths across all platforms in your Jest snapshots.
+
+> NOTE: All single backslashes ("\\") will be replaced by a forward slash ("/").
+ Also, any string that looks like a Windows drive letter ("C:\\") will be replaced by a forward slash ("/").
 
```

Comment:
Please review this and make sure it is clear / a wanted update.

Reply:
Yes, this makes perfect sense to be cross-platform and would be a good improvement.

I'm curious if the second part of this "NOTE" is needed though since we're replacing the beginning of any path with `<PROJECT_ROOT>`?

Reply:
Ah, just saw the use case for normalizing paths that are not in the project directory. It doesn't appear to be breaking, so I'm not opposed to the change. However, I am curious about the use case for it? Especially since switching machines or users could cause the path to break the snapshot test regardless (but because of the system environment, not the SUT)?

Reply:
I see what you are saying, that change came after when I was getting all of the tests fixed. As for a use case, I am not 100% sure. There are tests specifically comparing other non-project root paths to make sure the aren't updated with ``<PROJECT_ROOT>``, so the change felt needed, but maybe it wasn't. I'm okay with hard coding the non-project directory tests (as they were before) and not replacing windows drives, I don't think it will affect very many people (if any) either way. 

Thinking use cases made me realize this serializer should probably replace other paths such as [os.homedir()](https://nodejs.org/api/os.html#os_os_homedir) with ``<HOME_DIR>`` and [os.tmpdir()](https://nodejs.org/api/os.html#os_os_tmpdir) with ``<TEMP_DIR>``. I can put a PR in for those if you'd like.

For your reference:

One of the failing test examples that needed to replace C:\ with /
https://github.com/chrisblossom/jest-serializer-path/commit/2d6bd6eaa8c89e1a24a929378ac91a6752339c0f#diff-044bc223cd3309d59e7054541146894aR189

The failing test:
https://ci.appveyor.com/project/chrisblossom/jest-serializer-path/build/1.0.1/job/6rc6f6x67ovxr4y1

Failing test with just windows drive letters
https://ci.appveyor.com/project/chrisblossom/jest-serializer-path/build/1.0.2/job/u0bg16ploo0ctwob

Reply:
After looking at this again, I do like the idea of handling these types of path cases (in case they ever do come up); and it doesn't appear to introduce any breaking changes.

Replacing `os.homedir()` and `os.tmpdir()` especially sounds like a good next step if you care to send a PR. My original use case for this package was to snapshot my webpack configs. I could definitely see a webpack loader using `os.tmpdir()` that could sneak into a snapshot of that type.